core/lib/zksync_core/src/api_server/contract_verification/api_impl.rs

```diff
@@ -1,24 +1,44 @@
-use actix_web::{
-    web::{self, Json},
-    HttpResponse, Result as ActixResult,
+use std::sync::Arc;
+
+use axum::{
+    extract::{Path, State},
+    response::Response,
+    Json,
 };
 use serde::Serialize;
 use zksync_dal::CoreDal;
 use zksync_types::{contract_verification_api::VerificationIncomingRequest, Address};
 
 use super::{api_decl::RestApi, metrics::METRICS};
 
-fn ok_json(data: impl Serialize) -> ActixResult<HttpResponse> {
-    Ok(HttpResponse::Ok().json(data))
+fn ok_json(data: impl Serialize) -> Response<String> {
+    Response::builder()
+        .status(axum::http::StatusCode::OK)
+        .body(serde_json::to_string(&data).expect("Failed to serialize"))
+        .unwrap()
+}
+
+fn bad_request(message: &str) -> Response<String> {
+    Response::builder()
+        .status(axum::http::StatusCode::BAD_REQUEST)
+        .body(message.to_string())
+        .unwrap()
+}
+
+fn not_found() -> Response<String> {
+    Response::builder()
+        .status(axum::http::StatusCode::NOT_FOUND)
+        .body(String::new())
+        .unwrap()
 }
```

Comment:
Kinda clunky, but that's how API worked with actix, and the goal for this PR was to switch the used library, and not to fully refactor.